app/assets/javascripts/models/beta/story.js

```diff
@@ -325,6 +325,34 @@ export const cloneStory = (story) => {
   };
 };
 
+export const mergeWithFetchedStories = (
+  currentStories,
+  fetchedStories,
+  PastStoryIds
```

Comment:
Can we use camelCase here?

```suggestion
  pastStoryIds
```